src/hamiltonian.jl

```diff
@@ -42,8 +46,8 @@ phasepoint(
     h::Hamiltonian,
     θ::T,
     r::T;
-    ℓπ = DualValue(neg_energy(h, r, θ), ∂H∂θ(h, θ)),
-    ℓκ = DualValue(neg_energy(h, θ), ∂H∂r(h, r))
+    ℓπ=DualValue(∂H∂θ(h, θ)...),
+    ℓκ=DualValue(neg_energy(h, r, θ), ∂H∂r(h, r))
```

Comment:
@yebai Caught a bug here. Luckily the only usecase is to take a sum of these two values.

Reply:
nice catch!